pom.xml

```diff
@@ -284,6 +284,17 @@ THE SOFTWARE.
             <scope>test</scope>
         </dependency>
     </dependencies>
+    <dependencyManagement>
+        <dependencies>
+            <!-- TODO: Remove this when https://github.com/jenkinsci/subversion-plugin/pull/240 is released -->
+            <dependency>
+                <groupId>org.jenkins-ci.plugins</groupId>
+                <artifactId>mapdb-api</artifactId>
+                <version>1.0.9.0</version>
+                <scope>test</scope>
+            </dependency>
+        </dependencies>
+    </dependencyManagement>
```

Comment:
Try just updating the `subversion` dep to `2.13.0`.

Reply:
jenkinsci/workflow-multibranch-plugin#91